run-checks

```diff
@@ -105,26 +108,41 @@ if [ "$STATIC" = 1 ]; then
 
     # go vet
     echo Running vet
-    for pkg in $(go list ./... | grep -v '/vendor/' ); do
-        go vet $pkg
-    done
+    go list ./... | grep -v '/vendor/' | xargs go vet
```

Comment:
xargs only creates commandlines that "fit"; check `xargs --limits`. It'll launch a new `go vet` if it fills up (you can change this to instead abort if it fills up with `-x`, iirc)